src/index.html

```diff
@@ -1,12 +1,62 @@
 <!DOCTYPE html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+
+    <main>
+      <section
+        class="cal cal--month-length-30 cal--start-day-mon"
+      >
+
+        <div class="cal__day"></div>
```

Comment:
```suggestion
      >
        <div class="calendar__day"></div>
```
Don't leave newlines between parent and child elements. 
Also, it's better not to shorten block name so much